src/chain/index.ts

```diff
@@ -109,8 +120,8 @@ export class BeaconChain extends EventEmitter {
     // TODO: implement
     
     // The node's Unix time is greater than or equal to state.genesis_time + (block.slot - GENESIS_SLOT) * SECONDS_PER_SLOT.
-    const stateSlotTime = state.genesisTime.add(block.slot.sub(GENESIS_SLOT).muln(SECONDS_PER_SLOT));
-    if (!(new BN(Date.now())).gte(stateSlotTime)) {
+    const stateSlotTime = state.genesisTime + (block.slot - GENESIS_SLOT) * SECONDS_PER_SLOT;
```

Comment:
`const stateSlotTime = state.genesisTime + ((block.slot - GENESIS_SLOT) * SECONDS_PER_SLOT);`
I think you made a mistake with brackets

Reply:
should be the same order of operations ;) I can add the parens for more readability though.